src/enhancers/withComputedWidth.js

```diff
@@ -35,16 +35,17 @@ export default function withComputedWidth(BaseComponent) {
     handleResize = throttle(() => this.updateWidth(), 500)
 
     updateWidth() {
-      if (this.divRef.current && !this.props.width) {
+      if (this.divRef.current) {
         const width = this.divRef.current.clientWidth
         this.setState(prevState => (prevState.width === width ? null : {width}))
       }
     }
 
     render() {
+      const width = this.props.width != null ? this.props.width : this.state.width
```

Comment:
The Airbnb config [allows](https://github.com/airbnb/javascript/blob/5e260928b65f48c895b21d49ea0aa752df2e4d80/packages/eslint-config-airbnb-base/rules/best-practices.js#L40) `== null` as special case because it's the only practical way to check against `null` and `undefined` without incurring the cost of a function invocation. I agree that `isNil` is clearer, but as a stylistic choice I'd rather stay closer to the Airbnb config.

Reply:
Cool, that makes sense!